packages/cozy-procedures/package.json

```diff
@@ -21,12 +25,14 @@
   },
   "devDependencies": {
     "@babel/cli": "7.4.4",
-    "cozy-ui": "^20.18.0",
+    "@material-ui/core": "3.9.3",
+    "cozy-ui": "^20.24.0",
     "enzyme-to-json": "3.3.5",
     "jest": "24.8.0"
   },
   "peerDependencies": {
-    "cozy-ui": "^20.18.0"
+    "@material-ui/core": "3.9.3",
```

Comment:
Isn't it strange that `@material-ui/core` is in deps, peerDeps and devDeps ?

Reply:
Solved now!